.github/workflows/tests-karma-dashboard.yml

```
@@ -71,3 +101,28 @@ jobs:
         with:
           file: build/logs/karma-coverage/dashboard/lcov.info
           flags: karmatests
+
+  percy:
+    name: Percy
+    runs-on: ubuntu-latest
+    timeout-minutes: 5
+    if: always() && github.event.pull_request.draft == false
+    needs: [karma, nonce]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup Node
+        uses: actions/setup-node@v2.5.1
+        with:
+          node-version-file: '.nvmrc'
+          cache: npm
+
+      - name: Install dependencies
+        run: npm install @percy/cli
+
+      - name: Finalize Percy build
+        run: npx percy build:finalize
+        env:
+          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN }}
```

```suggestion
          PERCY_TOKEN: ${{ secrets.PERCY_TOKEN_DASHBOARD }}
```

This will send dashboard karma snapshots to a new project that I've just set up on Percy.